tests/assets/detect-touch.html

```diff
@@ -1,12 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Removing since the upstream test got removed: https://github.com/Modernizr/Modernizr/pull/2432. Updating the usages to just check for `'ontouchstart' in window` like it did internally.